odoo/tools/mail.py

```diff
@@ -536,6 +536,23 @@ def email_split_and_format(text):
         return []
     return [formataddr((name, email)) for (name, email) in email_split_tuples(text)]
 
+
+def email_domain(email):
+    """Return the domain of the given email."""
+    if not email:
+        return
+    *_, domain = email.rpartition('@')
+    return domain
+
+
+def email_domain_normalize(email_domain):
```

Comment:
Can't it be included in email_domain_extract (with maybe a parameter 'normalize=False/True') ?